### PR TITLE
Lab02

### PR DESCRIPTION
**Pull request by Bravelemming**

### CODE REVIEW

File: basics/src/test/java/basics/LibraryTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        assertNotNull("roll should return 'true'", classUnderTest.roll(1));
+    }
+
+    @Test
+    public void rollLengthIsCorrect() {
+        Library classUnderTest = new Library();
+        assertTrue("roll should have a length of at least n", classUnderTest.roll(1).length == 1);
+    }
+
+    @Test
```
Comment (Tw1stedRain) [inline basics/src/test/java/basics/LibraryTest.java:23]: in the future, id like to see more tests for each method. test some edge cases or at the very least test each method a couple times so i know its working completely and not just working in that single case that was tested
